CMakeLists.txt

```diff
@@ -230,6 +230,7 @@ add_custom_target(tbb_def_files DEPENDS tbb.def tbbmalloc.def)
 # TBB library
 if (TBB_BUILD_STATIC)
   add_library(tbb_static STATIC ${tbb_src})
+  target_include_directories(tbb_static PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include")
```

Comment:
It might be better to make this `INTERFACE` instead of `PUBLIC` as it is clearly not needed in the building of the library itself.